app/src/utils/url.js

```diff
@@ -0,0 +1,17 @@
+import * as apiCfg from "../config/apiConfig";
+
+export function getQueryParam(queryUrl) {
```

Comment:
these two fns are probably worth getting a couple quick and simple test cases into place for